### PR TITLE
Added the BC Player, updated the copyright year

### PR DESCRIPTION
**Pull request by philcluff**

### CODE REVIEW

File: bc-player.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>Phil's Players - Brightcove Player</title>
+
+  <!-- Bootstrap -->
+  <link href="css/bootstrap.min.css" rel="stylesheet">
+  <link href="css/ie10-viewport-bug-workaround.css" rel="stylesheet">
+  <link href="css/jumbotron-narrow.css" rel="stylesheet">
+
+  <link rel="stylesheet" type="text/css" href="css/videojs-contrib-dash.css"></link>
```
Comment (philcluff) [inline bc-player.html:15]: Should add a stylesheet for just for the BC player, or inline the one style which is in here anyway.

File: bc-player.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      player.src({
+        src: form['manifest'],
+        type: 'application/dash+xml'
+      });
+
+      player.play();
+      event.preventDefault();
+    });
+
+    $( "#demo" ).click(function() {
```
Comment (philcluff) [inline bc-player.html:69]: Maybe should use the MS CENC example as a demo URL here?

File: bc-player.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      </form>
+    </div>
+
+    <div class="jumbotron">
+      <video id="example" width="600" height="300" data-account="4276901753001" data-player="edae5cc2-925e-4679-9984-89f75c7ddc31" data-embed="default" class="video-js vjs-default-skin center-block" controls></video>
+      <script src="//players.brightcove.net/4276901753001/edae5cc2-925e-4679-9984-89f75c7ddc31_default/index.min.js"></script>
+    </div>
+
+    <div class="jumbotron">
+      <p>This player is The Brightcove Player with all DRM enabled.<br><a href="http://todo">More details can be found here.</a></p>
```
Comment (philcluff) [inline bc-player.html]: Fill in this TODO

File: bc-player.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+      <p><small>This should play CENC content everywhere, using the Brightcove special sauce for setting license URLs, <a href="http://todo">as documented here.</a></small></p>
```
Comment (stuarthicks) [inline bc-player.html]: Another TODO here.

File: bc-player.html [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+    <script>
+    var player = videojs('example');
+    $( "#playform" ).submit(function( event ) {
+      var form = {};
+      $.each($('#playform').serializeArray(), function(_, kv) {
+        form[kv.name] = kv.value;
+      });
+
+      console.log($( "#manifest" ).val());
```
Comment (thomshutt) [inline bc-player.html]: Don't forget to actually use this

**Pull request merged**